tfx/components/evaluator/executor.py

```diff
@@ -213,9 +235,7 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
              slice_spec=slice_spec,
              tensor_adapter_config=tensor_adapter_config))
       else:
```

Comment:
this 215 else branch is the same as 204 else branch:
so no need to have 

```
if _USE_TFXIO:
  if tfma.is_batched_input(eval_shared_model, eval_config):
  else:
else:
```

it can be just

```
if _USE_TFXIO and tfma.is_batched_input(eval_shared_model, eval_config):
else:
```

Reply:
I already removed the first else, so it's like this now. I thought it will still be 2 lines so I didn't use `and` to combine _USE_TFXIO and is_batched_input.

```
if _USE_TFXIO:
  if tfma.is_batched_input(eval_shared_model, eval_config):
else:
```

Reply:
the tfma.ExtractEvaluateAndWriteResults call can be dedup too

```
examples_list = []
tensor_adapter_config = None
if xxx:
  tensor_adapter_config = xxx
  examples_list=xxx
else:
  examples_list=xxx

tfma.ExtractEvaluateAndWriteResults(...)
```

Reply:
ah i see, yes, this is even better, i updated it